components/activity_log_modal/activity_log_modal.jsx

```diff
@@ -8,13 +8,17 @@ import {Modal} from 'react-bootstrap';
 import {FormattedDate, FormattedMessage, FormattedTime} from 'react-intl';
 import {General} from 'mattermost-redux/constants';
 
-import UserStore from 'stores/user_store.jsx';
 import {getMonthLong} from 'utils/i18n';
 import * as Utils from 'utils/utils.jsx';
```

Comment:
Can you change this so that we only import what we need for this component? Like:
```
import {isMobile, localizeMessage} from 'utils/utils.jsx';
```